mat/vector_test.go

```diff
@@ -307,12 +307,12 @@ func TestVecDenseAdd(t *testing.T) {
 		},
 		{
 			a:    NewVecDense(3, []float64{0, 1, 2}),
-			b:    NewDense(3, 1, []float64{0, 2, 3}).ColView(0),
+			b:    NewDense(3, 1, []float64{0, 2, 3}).ColView(0).(*VecDense),
```

Comment:
ColViewVecDense (and below)

Reply:
I think this depends on what is being tested here; I would prefer this to be assigned to a `Vector` field and have the assertion in one place at L320 so we remember to remove it when `AddVec` is changed to accept `Vector` arguments.